tests/browser/test_admin_project_access.py

```
@@ -19,25 +21,40 @@
+    """Accept the current TOS for a user."""
+    tos = TermsOfService.objects.filter(is_active=True).first()
+    if tos:
+        TermsOfServiceAcceptance.objects.get_or_create(
+            user=user,
+            tos_version=tos,
+            defaults={"ip_address": "127.0.0.1"},
+        )
+
+
 @pytest.fixture
 def owner(db):
```

Ruff (0.14.8)

Unused function argument: `db`

(ARG001)